salt/orchestrate/logging/configure_curator.sls

```diff
@@ -0,0 +1,38 @@
+# Elasticsearch Curator is used for elasticsearch snapshots
+install_elasticsearch_curator:
+  pip.installed:
+    - name: elasticsearch-curator
+    - require:
+      - salt: build_logging_nodes
+
+put_elasticsearch_snapshot_config:
+  http.query:
+    - name: http://elasticsearch.service.consul:9200/_snapshot/s3_repository/
+    - data: '
```

Comment:
Rather than quoting the whole thing you could use the `|` or `>-` operators for multiline YAML.